### PR TITLE
Implement Or in IlBuilder

### PR DESCRIPTION
**Pull request by ShuyuLi-ca**
In order to use `Or`, implement `Or` in `IlBuilder`.

Signed-off-by: Shuyu Li <shuyuli@ca.ibm.com>

### CODE REVIEW

**Comment by mstoodle** [comment]
Looks good to me. Thanks @ShuyuLi-ca !

File: compiler/ilgen/IlBuilder.cpp [hunk 1]
```diff
@@ -1263,6 +1263,15 @@ IlBuilder::And(TR::IlValue *left, TR::IlValue *right)
    }
 
 TR::IlValue *
+IlBuilder::Or(TR::IlValue *left, TR::IlValue *right)
+   {
+   TR::IlValue *returnValue=binaryOpFromOpMap(TR::ILOpCode::orOpCode, left, right);
+   TraceIL("IlBuilder[ %p ]::%d is Or %d || %d\n", this, returnValue->getCPIndex(), left->getCPIndex(), right->getCPIndex());
```
Comment (ymanton) [inline compiler/ilgen/IlBuilder.cpp]: Should be `|` not `||`.

Comment (ShuyuLi-ca) [inline compiler/ilgen/IlBuilder.cpp]: Sorry about this. I just fixed it. Thanks. :)

**Comment by 0xdaryl** [comment]
Ready to merge @mstoodle  ?

**Comment by mstoodle** [comment]
Thanks for the updates, @ShuyuLi-ca (and for the poke, @0xdaryl :) )

**Pull request merged**